docs/tutorials/ad-hoc-simulation/planning-simulation.md

```diff
@@ -2,81 +2,134 @@
+![after-autoware-launch](images/planning/lane-following/after-autoware-launch.png)
 
-3. Set a goal pose for the ego vehicle.
+#### 2. Add Autoware State Panel
 
-   - a) Click the `2D Nav Goal` button in the toolbar, or hit the `G` key
-   - b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose.
+This panel is useful when running planning simulations. To add the panel, click `Panels -> Add new panel`, select `AutowareStatePanel`, and then click `OK`.
 
-   ![set-goal-pose](images/planning/lane-following/set-goal-pose.png)
+![after-autoware-launch](images/planning/lane-following/open-autoware-state-panel.png)
 
-4. To place a dummy object such as a pedestrian, click the `2D Dummy Pedestrian` button in the toolbar.
+#### 3. Set an initial pose for the ego vehicle
 
-   ![set-dummy-pedestrian](images/planning/lane-following/set-dummy-pedestrian.png)
+![set-initial-pose](images/planning/lane-following/set-initial-pose.png)
 
-5. Engage the ego vehicle.
+a) Click the `2D Pose estimate` button in the toolbar, or hit the `P` key.
 
-   ```bash
-   source ~/autoware/install/setup.bash
-   ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
-   ```
+b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the initial pose. An image representing the vehicle should now be displayed.
 
-   ![start-driving](images/planning/lane-following/start-driving.png)
+!!! warning
 
-6. To delete objects, click the `Delete All Objects` button in the toolbar.
+    Remember to set the initial pose of the car in the same direction as the lane.
 
-   ![delete-objects](images/planning/lane-following/delete-objects.png)
+    To confirm the direction of the lane, check the arrowheads displayed on the map.
 
-7. After that, the vehicle will restart driving and reach the goal.
+#### 4. Set a goal pose for the ego vehicle
 
-   ![restart-driving](images/planning/lane-following/restart-driving.png)
+a) Click the `2D Goal Pose` button in the toolbar, or hit the `G` key.
 
-   ![reach-goal](images/planning/lane-following/reach-goal.png)
+b) In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose. If done correctly, you will see a planned path from initial pose to goal pose.
 
-## How to simulate parking maneuvers
+![set-goal-pose](images/planning/lane-following/set-goal-pose.png)
+
+#### 5. Engage the ego vehicle
+
+Now you can start the ego vehicle driving by clicking the `Engage` button in `AutowareStatePanel`. Alteratively, you can manually engage the vehicle by running the following command:
+
+```bash
+source ~/autoware/install/setup.bash
+ros2 topic pub /autoware/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
+```
+
+![start-driving](images/planning/lane-following/engage-and-start-planning.png)
+
+### Parking maneuvers
```

Comment:
[IMO] Using the words `Lane driving scenario` and `Parking scenario` might be better?

![image](https://user-images.githubusercontent.com/31987104/172384778-d027b2b3-c8d1-4b7c-9be9-2a52875a5d76.png)

Reply:
Oops thanks !